plugins/card_fetcher.py

```diff
@@ -124,7 +124,7 @@ def handle_message(self, msg):
 
 
 def get_card_obj(cardname):
-    r = requests.get('http://api.mtgdb.info/cards/{}'.format(cardname))
+    r = requests.get('http://api.mtgdb.info/cards/{}'.format(cardname.replace(':', '')))
 
     if (not r.json()) or (r.status_code is not requests.codes.ok):
```

Comment:
as discussed: reverse order of these so short circuiting takes care of a bad request before trying to parse json